cypress/integration/examples/connectors.spec.js

```diff
@@ -44,14 +44,55 @@ context('Connectors', () => {
     })
   })
 
-  it('.then() - invoke a callback function with the current subject', () => {
-    // https://on.cypress.io/then
-    cy.get('.connectors-list > li')
-      .then(($lis) => {
-        expect($lis, '3 items').to.have.length(3)
-        expect($lis.eq(0), 'first item').to.contain('Walk the dog')
-        expect($lis.eq(1), 'second item').to.contain('Feed the cat')
-        expect($lis.eq(2), 'third item').to.contain('Write JavaScript')
+  describe('.then()', () => {
+    it('invokes a callback function with the current subject', () => {
+      // https://on.cypress.io/then
+      cy.get('.connectors-list > li')
+        .then(($lis) => {
+          expect($lis, '3 items').to.have.length(3)
+          expect($lis.eq(0), 'first item').to.contain('Walk the dog')
+          expect($lis.eq(1), 'second item').to.contain('Feed the cat')
+          expect($lis.eq(2), 'third item').to.contain('Write JavaScript')
+        })
+    })
+
+    it('yields the returned value to the next command', () => {
+      cy.wrap(1)
+        .then((a) => {
```

Comment:
I'd prefer `a` & `b` to be named `num`, is less confusing

Reply:
I have changed the variable names to be `num` and copied the source into HTML